lib/firefox.js

```diff
@@ -113,6 +114,17 @@ function runFirefox (manifest, options) {
 
     function killFirefox () {
       firefox.kill();
+      var tries = 0;
+      // Windows release lock on files moments after the process ends
+      while (fs.existsSync(profile) && tries < 1000) {
```

Comment:
it looks like this is not guaranteed to work any better than things do now.  We should see if there is some alternative, like a package on npm which will handle removing files across platforms well.